src/Mod/BIM/Arch.py

```diff
@@ -788,7 +788,22 @@ def makeSpace(objects=None,baseobj=None,name=None):
     if objects:
         if not isinstance(objects,list):
             objects = [objects]
-        if len(objects) == 1:
+        isSingleObject = len(objects) == 1
+        try:
+            if hasattr(objects[0],"HasSubObjects"):
+                # Selection set
+                # [<SelectionObject>]
+                hasNoBoundaries = not objects[0].HasSubObjects
+            else:
+                # Single object with subojects. If it has only
```

Comment:
Talking about a single object here is confusing. You are testing the first item in a list. There is a typo: 'subojects'.

Reply:
```suggestion
                # Single object with subobjects. If it has only
```
Thanks for the review. Typo fixed in the suggestion.

> Talking about a single object here is confusing. You are testing the first item in a list.

Unless I'm mistaken (and I might well be), at this point only one-item lists will be encountered. That's the case where `makeSpace` is passed either:
- `[ (<Part::PartFeature>, ["Face1", ...]) ]` (or `(<Part::PartFeature>, ["Face1", ...])`, which is converted to a list on L789 
- `[ <Part::PartFeature> ]` (or `<Part::PartFeature>`, which is converted to a list on L789